atat_internal_api.yaml

```diff
@@ -2038,6 +2038,37 @@ components:
   #          type: string
   #        lastName:
   #          type: string
+    CspInvocation:
+      required:
+        - method
+        - endpoint
+        - payload
```

Comment:
Are `headers` not required?

Reply:
Nope - I didn't want to impose anything on the process but in theory we might need to pass some headers in some day that might require some dynamic calculation or other reason for including in this object (vs just hard-coding it in the invocation Lambda).